diff -u parsimonious_orig/exceptions.py parsimonious/exceptions.py
--- parsimonious_orig/exceptions.py	2016-11-11 19:03:49.939272570 -0700
+++ parsimonious/exceptions.py	2016-11-16 14:22:43.500052783 -0700
@@ -1,6 +1,6 @@
 from six import text_type, python_2_unicode_compatible
 
-from parsimonious.utils import StrAndRepr
+from contractor.tscript.parsimonious.utils import StrAndRepr
 
 
 @python_2_unicode_compatible
diff -u parsimonious_orig/expressions.py parsimonious/expressions.py
--- parsimonious_orig/expressions.py	2016-11-11 19:03:49.939272570 -0700
+++ parsimonious/expressions.py	2016-11-16 14:21:29.055997823 -0700
@@ -12,9 +12,9 @@
 from six import integer_types, python_2_unicode_compatible
 from six.moves import range
 
-from parsimonious.exceptions import ParseError, IncompleteParseError
-from parsimonious.nodes import Node, RegexNode
-from parsimonious.utils import StrAndRepr
+from contractor.tscript.parsimonious.exceptions import ParseError, IncompleteParseError
+from contractor.tscript.parsimonious.nodes import Node, RegexNode
+from contractor.tscript.parsimonious.utils import StrAndRepr
 
 MARKER = object()
 
diff -u parsimonious_orig/grammar.py parsimonious/grammar.py
--- parsimonious_orig/grammar.py	2016-11-11 19:03:49.939272570 -0700
+++ parsimonious/grammar.py	2016-11-16 14:21:47.316010211 -0700
@@ -11,12 +11,12 @@
 from six import (text_type, iterkeys, itervalues, iteritems,
     python_2_unicode_compatible, PY2)
 
-from parsimonious.exceptions import BadGrammar, UndefinedLabel
-from parsimonious.expressions import (Literal, Regex, Sequence, OneOf,
+from contractor.tscript.parsimonious.exceptions import BadGrammar, UndefinedLabel
+from contractor.tscript.parsimonious.expressions import (Literal, Regex, Sequence, OneOf,
     Lookahead, Optional, ZeroOrMore, OneOrMore, Not, TokenMatcher,
     expression)
-from parsimonious.nodes import NodeVisitor
-from parsimonious.utils import StrAndRepr, evaluate_string
+from contractor.tscript.parsimonious.nodes import NodeVisitor
+from contractor.tscript.parsimonious.utils import StrAndRepr, evaluate_string
 
 @python_2_unicode_compatible
 class Grammar(StrAndRepr, Mapping):
diff -u parsimonious_orig/__init__.py parsimonious/__init__.py
--- parsimonious_orig/__init__.py	2016-11-11 19:03:49.939272570 -0700
+++ parsimonious/__init__.py	2016-11-16 14:23:24.224087541 -0700
@@ -3,8 +3,8 @@
 Things may move around in modules deeper than this one.
 
 """
-from parsimonious.exceptions import (ParseError, IncompleteParseError,
+from contractor.tscript.parsimonious.exceptions import (ParseError, IncompleteParseError,
                                      VisitationError, UndefinedLabel,
                                      BadGrammar)
-from parsimonious.grammar import Grammar, TokenGrammar
-from parsimonious.nodes import NodeVisitor, VisitationError, rule
+from contractor.tscript.parsimonious.grammar import Grammar, TokenGrammar
+from contractor.tscript.parsimonious.nodes import NodeVisitor, VisitationError, rule
diff -u parsimonious_orig/nodes.py parsimonious/nodes.py
--- parsimonious_orig/nodes.py	2016-11-11 19:03:49.939272570 -0700
+++ parsimonious/nodes.py	2016-11-16 14:22:04.032022191 -0700
@@ -12,8 +12,8 @@
 from six import reraise, python_2_unicode_compatible, with_metaclass, \
     iteritems
 
-from parsimonious.exceptions import VisitationError, UndefinedLabel
-from parsimonious.utils import StrAndRepr
+from contractor.tscript.parsimonious.exceptions import VisitationError, UndefinedLabel
+from contractor.tscript.parsimonious.utils import StrAndRepr
 
 
 @python_2_unicode_compatible
Only in parsimonious: __pycache__
